sdk/core/Microsoft.Azure.Core.Spatial/tests/Microsoft.Azure.Core.Spatial.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <!-- We don't care about XML doc comments on test methods -->
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+  </PropertyGroup>
+
+  <!-- Reference the Client Library -->
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.Azure.Core.Spatial.csproj" />
+  </ItemGroup>
+
+  <!-- Add Test References -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="nunit" />
+    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Moq" />
+    <PackageReference Include="System.Memory" />
+    <ProjectReference Include="$(AzureCoreTestFramework)" />
+  </ItemGroup>
+
+  <!-- Pull in Shared Source from Azure.Core -->
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)HashCodeBuilder.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
Why do we need HashCodeBuilder in tests?

Reply:
It was used for an `IEqualityComparer<T>` I wrote but no longer use. I forgot to remove it.